hydrolib/core/io/mdu/models.py

```diff
@@ -502,12 +503,169 @@ def is_intermediate_link(self) -> bool:
         return True
 
 
+class Calibration(INIBasedModel):
+    """
+    The `[Calibration]` section in an MDU file.
+
+    This model is typically referenced under [FMModel][hydrolib.core.io.mdu.models.FMModel]`.calibration`.
+
+    All lowercased attributes match with the [Calibration] input as described in
+    [UM Sec.A.3](https://content.oss.deltares.nl/delft3d/manuals/D-Flow_FM_User_Manual_1D2D.pdf#section.A.3).
+    """
+
+    _header: Literal["Calibration"] = "Calibration"
+    usecalibration: bool = Field(0, alias="UseCalibration")
+    definitionfile: Optional[Path] = Field(None, alias="DefinitionFile")
```

Comment:
We have to remember to change `Optional[Path]` -> `DiskOnlyFileModel` after #281 is closed?

Reply:
Yes! :)